Formula/leela-zero.rb

```diff
@@ -0,0 +1,37 @@
+class LeelaZero < Formula
+  desc "Neural Network Go engine with no human-provided knowledge"
+  homepage "https://zero.sjeng.org/"
```

Comment:
Not reachable over HTTPS. I suggest we use the github page: https://github.com/gcp/leela-zero